doc/conf.py

```diff
@@ -94,9 +96,22 @@
     "doc_module": "harmonica",
     # Insert links to documentation of objects in the examples
     "reference_url": {"harmonica": None},
+    # Add pyvista to the image scrapers
+    "image_scrapers": ("pyvista"),
```

Comment:
This would actually remove the default MPL scraper. Be sure to include it as well

```suggestion
    "image_scrapers": ("pyvista", "matplotlib"),
```

Reply:
Thanks for that! I was just copy-pasting your docs.